package.json

```diff
@@ -10,30 +10,32 @@
   "license": "MIT",
   "private": true,
   "scripts": {
-    "build": "gulp",
-    "watch": "gulp watch"
+    "dev": "node ./webpack.server.js",
+    "build": "rimraf ./dist && webpack --config ./webpack/webpack.prod.config.js",
+    "preview": "http-server -p 3001 ./dist -c-1 -o"
   },
   "dependencies": {
     "clipboard": "^1.5.12",
     "react": "^15.3.2",
     "react-dom": "^15.3.2"
   },
   "devDependencies": {
+    "babel-core": "^6.18.0",
+    "babel-loader": "^6.2.7",
     "babel-plugin-transform-class-properties": "^6.16.0",
-    "babel-preset-es2015": "^6.16.0",
+    "babel-preset-es2015": "^6.18.0",
     "babel-preset-react": "^6.16.0",
-    "babel-register": "^6.16.3",
-    "babelify": "^7.3.0",
-    "browser-sync": "^2.17.0",
-    "browserify": "^13.1.0",
-    "del": "^2.2.2",
-    "gulp": "github:gulpjs/gulp#4.0",
-    "gulp-cssnano": "^2.1.2",
-    "gulp-gh-pages": "^0.5.4",
-    "gulp-htmlmin": "^3.0.0",
-    "gulp-sass": "^2.3.2",
-    "gulp-uglify": "^2.0.0",
-    "vinyl-buffer": "^1.0.0",
-    "vinyl-source-stream": "^1.1.0"
+    "extract-text-webpack-plugin": "v2.0.0-beta.4",
+    "html-webpack-plugin": "^2.24.0",
+    "http-server": "^0.9.0",
+    "node-sass": "^3.10.1",
+    "offline-plugin": "^3.4.2",
+    "react-hot-loader": "^3.0.0-beta.6",
+    "rimraf": "^2.5.4",
+    "sass-loader": "^4.0.2",
+    "webpack": "2.1.0-beta.22",
+    "webpack-dev-server": "2.1.0-beta.9",
+    "webpack-merge": "^0.15.0",
```

Comment:
Looks like you rolled your own way of merging webpack configs, so I guess this isn't required.